samples/peripheral/radio_test/src/radio_test.c

```diff
@@ -411,11 +411,27 @@ static void radio_disable(void)
 #endif /* CONFIG_FEM */
 }
 
+static void radio_mode_set(NRF_RADIO_Type *reg, nrf_radio_mode_t mode)
+{
+	/* Workaround for Errata 117 */
+	#if CONFIG_SOC_NRF5340_CPUNET_QKAA
```

Comment:
I suggest to use `NRF53_ERRATA_117_ENABLE_WORKAROUND` definitions instead of `CONFIG_SOC_NRF5340_CPUNET_QKAA`. Also it would be good to put errata code in the separate function.

```
#if NRF53_ERRATA_117_ENABLE_WORKAROUND
void errata_117(void) {
    /* Do errata here */
}
#else
void errata_117(void) {
}
#endif
```